rflib/main/default/classes/rflib_ApplicationEventArchiver.cls

```diff
@@ -50,16 +50,23 @@ public inherited sharing class rflib_ApplicationEventArchiver implements Schedul
     public void execute(SchedulableContext ctx) {
         try {
             LOGGER.info('execute() invoked');
-
+    
             Date threshold = TODAY.addDays(rflib_GlobalSettings.daysToRetainApplicationEventsOrDefault * (-1));
-
-            List<rflib_Application_Event__c> applicationEvents = [
-                SELECT Id, Created_By_ID__c, Occurred_On__c, Event_Name__c, Related_Record_ID__c, Additional_Details__c
-                FROM rflib_Application_Event__c
-                WHERE Occurred_On__c < :threshold
-                LIMIT :rflib_GlobalSettings.archiveAppEventQueryLimitOrDefault
-            ];
-
+    
+            List<rflib_Application_Event__c> applicationEvents = rflib_SOQL.of(rflib_Application_Event__c.SObjectType)
```

Comment:
BTW.  `WITH USER_MODE` is a default FLS mode for all queries in SOQL Lib. It's automatically added.

You can:
- Add `.systemMode()` to your query if you need SYSTEM MODE which was default for that query
- Or change the default mode in SOQL Lib in Executor:

<img width="450" alt="image" src="https://github.com/user-attachments/assets/8e9551ad-af8a-4603-b6c1-ba90bd4d6057" />

Reply:
Thank you, I read about the modes but did not realize that user mode was the default. I like the fact to make it explicit and keep the default as user mode as this should be the default for using SOQL outside the library.